scripts/install/install.sh

```diff
@@ -21,11 +21,11 @@ target=${1-prod}
 
 if [[ $target == *"dev"* ]]; then
```

Comment:
ok .. this is good to know .... that the default is not dev is something to keep in mind ... what's the syntax?